src/hooks/wallets/wallets.ts

```diff
@@ -54,6 +56,8 @@ export const getAllWallets = (chain: ChainInfo): WalletInits => {
 
 export const isWalletSupported = (disabledWallets: string[], walletLabel: string): boolean => {
   const legacyWalletName = CGW_NAMES?.[walletLabel.toUpperCase() as WALLET_KEYS]
+  console.log('legacyWalletName', legacyWalletName)
+  console.log('isEnabled', !disabledWallets.includes(legacyWalletName || walletLabel))
```

Comment:
```suggestion
```

Reply:
Removed in 9eda3fe5521a1de371c30760c02dcce80875db3a